aspnetcore/tutorials/first-mvc-app/adding-model.md

```diff
@@ -102,6 +102,18 @@ The `Add-Migration` command creates code to create the initial database schema.
 
 The `Update-Database` command runs the `Up` method in the *Migrations/\<time-stamp>_InitialCreate.cs* file, which creates the database.
 
+You can perform the preceeding steps using the command-line interface (CLI) rather than the PMC:
+
+* Add [EF Core tooling](xref:data/ef-mvc/migrations#entity-framework-core-nuget-packages-for-migrations) to the *.csproj* file.
+* Run the following commands from the console (in the project directory):
+
+  ```console
```

Comment:
to put this under the Run the following commands bullet, the opening ``` is indented 2, but following lines and closing ``` are indented more